src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/ParserHelpers.cs

```diff
@@ -52,6 +52,27 @@ public static bool IsDigit(int i)
             return (uint)(i - '0') <= ('9' - '0');
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
do we have to force the inlining of this method? I would assume that JIT is going to do this even without it

Reply:
BTW I've used https://godbolt.org/z/imxPHx to see if we could use some magic bit operation to get the same effect, but it looks like no ;)

Reply:
The codegen turns this into three instructions: `add`, `cmp`, `jcc`. I'd say that's fairly efficient. :)